operations/app/src/modules/database/main.tf

```diff
@@ -0,0 +1,55 @@
+terraform {
+    required_version = ">= 0.14"
+}
+
+resource "azurerm_postgresql_server" "postgres_server" {
+  name = var.name
+  location = var.location
+  resource_group_name = var.resource_group
+
+  administrator_login = var.postgres_user
+  administrator_login_password = var.postgres_password
+
+  sku_name = "GP_Gen5_4"
+  version = "11"
+  storage_mb = 5120
+
+  auto_grow_enabled = (var.environment == "prod" ? true : false)
+
+  ssl_enforcement_enabled = true
+  ssl_minimal_tls_version_enforced = "TLS1_2"
+}
+
+resource "azurerm_postgresql_virtual_network_rule" "allow_public_subnet" {
+  name = "AllowPublicSubnet"
+  resource_group_name = var.resource_group
+  server_name = azurerm_postgresql_server.postgres_server.name
+  subnet_id = var.public_subnet_id
+}
+
+resource "azurerm_postgresql_virtual_network_rule" "allow_private_subnet" {
+  name = "AllowPrivateSubnet"
+  resource_group_name = var.resource_group
+  server_name = azurerm_postgresql_server.postgres_server.name
+  subnet_id = var.private_subnet_id
+}
+
+resource "azurerm_postgresql_database" "prime_data_hub_db" {
+  name = "prime_data_hub"
+  resource_group_name = var.resource_group
+  server_name = azurerm_postgresql_server.postgres_server.name
+  charset = "UTF8"
+  collation = "English_United States.1252"
+}
+
+resource "azurerm_postgresql_database" "metabase_db" {
```

Comment:
Could we add a [`lifecycle.prevent_destroy = true`](https://www.terraform.io/docs/language/meta-arguments/lifecycle.html) to this module? This contains data that would be damaging if accidentally ran a terraform destroy on it.

Reply:
Added